libsrc/lynx/ser/lynx-comlynx.s

```diff
@@ -108,17 +113,17 @@ SER_OPEN:
         stz     TxPtrIn
         stz     TxPtrOut
 
-        ; clock = 8 * 15625
-        lda     #%00011000
-        sta     TIM4CTLA
         ldy     #SER_PARAMS::BAUDRATE
         lda     (ptr1),y
 
+        ; Source period is 1 us
+        ldy     #%00011000  ; ENABLE_RELOAD|ENABLE_COUNT|AUD_1
```

Comment:
Why use here the direct value and not the symols?

Reply:
I will add and change more symbols in another pull request. Didn't want to introduce these yet, and instead do that when working on the timers. The comment is for later when the symbols are introduced.